scripts/common/prompts.sh

```diff
@@ -311,7 +311,7 @@ function prompt_for_private_ip() {
     fi
 
     if ! prompts_can_prompt ; then
-        bail "kurl.privateAddress required"
+        bail "kurl.privateAddress required. Try passing the private address to this script e.g. 'sudo ./install.sh private-address=1.2.3.4' or supply the privateAddress field to the kurl add-on."
```

Comment:
I wouldn't know what `privateAddress` is here. Can we say something about multiple NICs and needing to select one?

Reply:
Maybe we can re-use lines 318-321?